### ISSUE TITLE
macOS installation requirements missing requires brew/ports gnu-sed/gsed package

### ISSUE THREAD

**Issue by AshKetch-um** [avast-tl/retdec#257]
The retdec runtime uses `sed` with flags that are incompatible with the `sed` built into MacOS, however the [Run on MacOS](https://github.com/avast-tl/retdec#macos) instructions omit this requirement. 

I resolved this by installing the `gsed` (`sudo port install gsed`) package and then including the GNU symlink tree (`/opt/local/libexec/gnubin`) in my path before the built-in directories. A similar solution using brew's `gnu-sed` package should work as well.

**Environment**
MacOS 10.13.3
Mac Ports 
changeset: 31cf35fff9dce8de081a8705653f2e56430ce270

**Steps to Reproduce**
Download, Build and Install on MacOS
```
sudo port selfupdate
sudo port install cmake git perl5 python27 bison flex autoconf automake libtool coreutils wget doxygen upx graphviz

# See #98
export PATH=/opt/local/bin:/opt/local/sbin:/opt/local/libexec/gnubin:$PATH

git clone https://github.com/avast-tl/retdec
cd retdec
mkdir build && cd build
cmake .. -DCMAKE_INSTALLPREFIX=/opt/local/retdec
make -j6
sudo make install
```

Run against a simple test 32-bit Linux ELF binary
```
cd /opt/local/retdec/bin
bash ./retdec-decompiler.sh /tmp/test
```

**Expected Behavior**
Clean run
```
Running phase: emission of the target code [c] ( 0.03s )
Running phase: finalization ( 0.03s )
Running phase: cleanup ( 0.03s )

##### Done!
```

**Observed Behavior**
`sed` errors
```
Running phase: emission of the target code [c] ( 0.03s )
Running phase: finalization ( 0.03s )
Running phase: cleanup ( 0.03s )
sed: 1: "/private/tmp/test.c": undefined label 'mp/test.c'
sed: 1: "/private/tmp/test.c": undefined label 'mp/test.c'

##### Done!
```

```
user@host:/opt/local/retdec/bin:$ which sed
/usr/bin/sed
```

**Comment by s3rvac** [comment]
Thank you for the report. I have fixed this in commit 7b5289acda02cfb8fbbf3a618c5a57c3f9c5a2a2. In short, I have modified our `sed` commands so that they work even with the version of `sed` on macOS, so there is no need for installing `gsed`.

The reason why the commands failed is that when using the `-i` parameter of `sed` on macOS, you need to specify an extension of the backup file. If you do not want to create a backup file, you have to pass the empty string. On Linux, using just `-i` without any extension suffices. Nevertheless, there were also some other issues with the original `sed` commands, so I rewrote them. Another advantage of the new version is that both `sed` commands are now part of a single `sed` command.

**Comment by AshKetch-um** [comment]
Awesome, thank you!
